pandapower/plotting/__init__.py

```diff
@@ -5,6 +5,7 @@
 from pandapower.plotting.simple_plot import *
 from pandapower.plotting.plotly import *
 from pandapower.plotting.geo import *
+from pandapower.plotting.plotting_toolbox import *
```

Comment:
There are many rather internal functions in the plotting toolbox and it is not in the documentation. Thus, I'd suggest to not import plotting_toolbox right away with pandapower.plotting